webpack.config.common.js

```diff
@@ -150,7 +150,8 @@ module.exports = {
           to: `${dist}/min/core-crypto.wasm`,
         },
         // copying all static resources (audio, images, fonts...)
-        {from: 'resource', to: dist},
+        {from: 'src/page/browser.js', to: dist},
+        {from: 'resource', to: path.resolve(dist, 'browser.js')},
```

Comment:
do we need those 2 lines, I feel only the first one is needed, isn't it? 

Let's move the `browser.js` file to `${dist}/min` so that all the js are there in the end

Reply:
done